man/systemctl.xml

```diff
@@ -567,6 +567,21 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           <option>ExecStartPre=</option>, etc.) </para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><command>mount-image</command> <replaceable>UNIT</replaceable> <replaceable>PATH</replaceable> [<replaceable>PATH</replaceable> [<replaceable>PARTITION_NAME</replaceable><replaceable>MOUNT_OPTIONS</replaceable>]]</term>
```

Comment:
hmm, maybe change one PATH to IMAGE.

hmm, no colon between partition name and options?

Reply:
> hmm, no colon between partition name and options?

Yes, I figure it's not needed to complicate the syntax, since it's a command line rather than a config file, and can just take any extra args as mount options. Felt easier when typing.

Reply:
Changed to use colon as separator, PTAL